modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/info_panel/tab_content/VersionsTabContent.tsx

```diff
@@ -3,10 +3,110 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import React from 'react';
+import {Button} from '@clayui/core';
+import Label from '@clayui/label';
+import {dateUtils, fetch, sub} from 'frontend-js-web';
+import React, {useCallback, useContext, useEffect, useState} from 'react';
+
+import {IAssetObjectEntry} from '../../../structure_builder/types/AssetType';
+import {
+	AssetTypeInfoPanelContext,
+	IAssetTypeInfoPanelContext,
+} from '../context';
+
+const MAX_LIST_SIZE = 10;
 
 const VersionsTabContent = () => {
-	return <></>;
+	const {objectEntries = []}: IAssetTypeInfoPanelContext = useContext(
+		AssetTypeInfoPanelContext
+	);
+
+	const [versionedObjectEntries, setVersionedObjectEntries] = useState<{
+		count: number;
+		items: IAssetObjectEntry[];
+	}>({count: 0, items: []});
+
+	const getVersionedObjectEntries = useCallback(async () => {
+		const [
+			{
+				actions: {
+					versions: {href = ''},
+				},
+			},
+		] = objectEntries;
+
+		const response = await fetch(
+			`${href}?page=1&pageSize=${MAX_LIST_SIZE}`
+		);
+
+		if (response.ok) {
+			const {items, totalCount} = await response.json();
+
+			setVersionedObjectEntries({count: totalCount, items});
+		}
+	}, [objectEntries, setVersionedObjectEntries]);
+
+	useEffect(() => {
+		getVersionedObjectEntries();
+	}, [getVersionedObjectEntries]);
+
+	return (
+		<>
+			<ul className="list-group">
```

Comment:
Why are you using HTML when you can use this?
https://clayui.com/docs/components/list#quick-action-menu (without quick actions)

Docs here: https://liferay.atlassian.net/wiki/spaces/COMMERCE/pages/3840704685/CMS+2.0+-+Side+Panel+for+Asset+Types#Versions-Tab-(any-single-asset)